src/pages/MonthlyGood/LandingForm.vue

```diff
@@ -0,0 +1,126 @@
+<template>
+	<form @submit.prevent.stop="submit" novalidate>
+		<fieldset>
+			<label class="input-label" :class="{ 'error': $v.$invalid }" for="amount">
+				<kv-currency-input id="amount" :value="amount" @input="updateAmount" />
+				<ul class="validation-errors">
```

Comment:
Nitpick, this ul will always be on the dom even if no errors. Perhaps a `v-if="$v.$invalid"`.